### PR TITLE
[3] Connection Monitor Metrics

### PR DESCRIPTION
**Pull request by kriti-sc**
PR [1] Connection Monitor: https://github.com/Netflix/mantis/pull/536
PR [2] Test: https://github.com/Netflix/mantis/pull/550
PR [3] Metrics: https://github.com/Netflix/mantis/pull/551

### CODE REVIEW

File: mantis-server/mantis-server-worker-client/build.gradle [hunk 1]
```diff
@@ -18,6 +18,8 @@ dependencies {
     api project(":mantis-control-plane:mantis-control-plane-core")
     api project(":mantis-control-plane:mantis-control-plane-client")
 
+    implementation libraries.spectatorApi
```
Comment (calvin681) [inline mantis-server/mantis-server-worker-client/build.gradle:21]: @hmit should we still have spectator api?

Comment (kriti-sc) [reply mantis-server/mantis-server-worker-client/build.gradle:21]: Adding some context: the build was failing without this.

Comment (hmitnflx) [reply mantis-server/mantis-server-worker-client/build.gradle:21]: yes it's still needed

**Review by hmitnflx — COMMENTED** [review]
lgtm otherwise

**Review by calvin681 — APPROVED** [review]

**Pull request merged**